CONTRIBUTING.md

```diff
@@ -1,51 +1,20 @@
-# Building and Running Tests Locally
+# Contributing Guide
 
-To ensure that the Trace Agent works cross-platform, we use TypeScript build scripts located in the [`scripts`](scripts) directory. The entry point to these scripts is [`index.ts`](scripts/index.ts). The usage of this file is `ts-node -p ./scripts ./scripts/index.ts [command1] [...moreCommands]` (assuming that you are running in the repository root directory.)
+The Stackdriver Trace Agent is written in TypeScript. This means that when testing changes in your own application, you will need to run `npm run compile` to invoke the TypeScript compiler to compile your changes to JavaScript.
```

Comment:
`when testing your own application`? I am really confused by this. is this about testing the Trace Agent or the user's application. The latter may not be in TS...

Reply:
It was supposed to mean "when testing changes to your local copy of the Trace Agent linked in your own application".